app/index.html

```diff
@@ -13,7 +13,7 @@ <h1>Título</h1>
     <nav>
         <ul>
             <li><a href="pages/jean.html">Professor: Jean</a></li>
```

Comment:
```suggestion
            <li><a href="pages/jean.html">Professor: Jean</a></li>
```
Tem conflito na branch pois existem dois links no arquivo index.